packages/storybook/stories/components/ButtonGroup.stories.tsx

```diff
@@ -0,0 +1,113 @@
+import { ArgsTable, Description, Primary, PRIMARY_STORY, Stories } from '@storybook/addon-docs';
+import { Meta, StoryObj } from '@storybook/react';
+import { ButtonGroup } from '@utrecht/component-library-react/dist/css-module';
+import readme from '@utrecht/components/badge-counter/README.md?raw';
+import tokensDefinition from '@utrecht/components/badge-counter/tokens.json';
```

Comment:
The `readme` and `tokensDefinition` are incorrectly imported from `badge-counter`.